### PR TITLE
OpenAPI: fix key_backup

### PR DESCRIPTION
**Pull request by lukaslihotzki**

### CODE REVIEW

File: data/api/client-server/key_backup.yaml [hunk 1]
```diff
@@ -820,7 +820,7 @@ paths:
       summary: Retrieve the keys from the backup for a given room
       description: |-
         Retrieve the keys from the backup for a given room
-      operationId: getRoomKeysKeyRoomId
+      operationId: getRoomKeysKey
```
Comment (KitsuneRal) [inline data/api/client-server/key_backup.yaml:823]:
```suggestion
      operationId: getRoomKeys
```
I guess, the original author used the `Key` infix where `By` is used in other places of the spec (see, e.g. `directory.yaml` or `rooms.yaml`).

File: data/api/client-server/key_backup.yaml [hunk 2]
```diff
@@ -879,7 +879,7 @@ paths:
       summary: Delete a key from the backup
       description: |-
         Delete a key from the backup.
-      operationId: deleteRoomKeysKeyRoomId
+      operationId: deleteRoomKeysKey
```
Comment (KitsuneRal) [inline data/api/client-server/key_backup.yaml:882]:
```suggestion
      operationId: deleteRoomKeys
```

**Review by KitsuneRal — CHANGES_REQUESTED** [review]
First off, thanks for taking care of that - my code generator still doesn't use this file but eventually will.

Now, at the very least, those trailing `Key` things should not be there. And ideally, since you're at it, it would be great if you could align `operationId` naming throughout the file (it seems to be the only file in the API using `Key` instead of `By`). And if you go for it, `RoomIdSessionId` in another call seems extraneous, I don't think cross-room megolm sessions are a thing.

File: data/api/client-server/key_backup.yaml [hunk 3]
```diff
@@ -375,7 +375,7 @@ paths:
       summary: Store a key in the backup.
       description: |-
         Store a key in the backup.
-      operationId: postRoomKeysKeyRoomIdSessionId
+      operationId: postRoomKeysBySessionId
```
Comment (deepbluev7) [inline data/api/client-server/key_backup.yaml:378]: If you are already changing this, it might make sense to `s/post/put/` here :D

Comment (KitsuneRal) [reply data/api/client-server/key_backup.yaml:378]: Indeed, that should be `put` (here and in two more places, actually).

**Review by KitsuneRal — APPROVED** [review]
Awesome. Sorry for not bringing all requests in one package but could you supply a changelog here as well? There's no tangible change in the documentation from this PR but code that uses operationIds will definitely look different. Thanks.

**Review by KitsuneRal — APPROVED** [review]

**Pull request merged**